docs/pages/router/reference/authentication.mdx

```diff
@@ -394,4 +394,4 @@ export default function RootLayout() {
 
 ## Middleware
 
-Traditionally, websites may leverage some form of server-side redirection to protect routes. As of Expo Router v2, this library on the web currently only supports build-time static generation and has no support for custom middleware or serving. This can be added in the future to provide a more optimal web experience. In the meantime, authentication can be implemented by using client-side redirects and a loading state.
+Traditionally, websites may leverage some form of server-side redirection to protect routes. As of Expo Router v3.5, this library on the web currently only supports build-time static generation and has no support for custom middleware or serving. This can be added in the future to provide a more optimal web experience. In the meantime, authentication can be implemented by using client-side redirects and a loading state.
```

Comment:
```suggestion
Traditionally, websites may leverage some form of server-side redirection to protect routes. Expo Router on the web currently only supports build-time static generation and has no support for custom middleware or serving. This can be added in the future to provide a more optimal web experience. In the meantime, authentication can be implemented by using client-side redirects and a loading state.
```

We only document guides and API related info for Expo Router for the current on-going version (otherwise specifically mentioned). I understand this can be confusing, so let's remove it the version specification  here.